plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/cart-express-payment-block/edit.tsx

```diff
@@ -24,15 +35,34 @@ export const Edit = ( {
 				hasExpressPaymentMethods,
 		} ),
 	} );
-	const { className } = attributes;
+
+	useEffect( () => {
+		const { showButtonStyles, buttonHeight, buttonBorderRadius } =
```

Comment:
Why are we reading this inside useEffect? Feels kinda weird and counterintuitive.

Reply:
We need a `useEffect` here because we want to read the settings when the component is first mounted, and sync them to the attributes. Attributes are then accessed normally by the component.

Reply:
Not asking about the need of useEffect, it make sense, I'm wondering why we're accessing settings inside a useEffect.

Reply:
I've removed the useEffects now as I've got rid of the option and we're syncing directly between cart and checkout